apps/communications/dialer/test/unit/call_log_test.js

```diff
@@ -29,6 +29,9 @@ require('/shared/test/unit/mocks/mock_image.js');
 require('/shared/test/unit/mocks/mock_sim_settings_helper.js');
 require('/shared/test/unit/mocks/dialer/mock_contacts.js');
 require('/shared/test/unit/mocks/mock_mozContacts.js');
+require('/shared/js/fb/fb_request.js');
```

Comment:
Yikes, we shouldn't have to do this, but I don't see a mock for it. We should file a bug against the Contacts app to create a mock for this.